Jof 1898197 minor edits (#142)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_setting-up-grafana-to-host-the-dashboard.adoc

+
[source,bash]
----
$ oc project service-telemetry
----

. To retrieve the default username and password, describe the Grafana object using the `oc describe` command:
+
[source,bash]
----
$ oc describe grafana default
----
+

> TIP: You can set the admin username and password through the `ServiceTelemetry` object using `graphing.grafana.adminUser` and `graphing.grafana.adminPassword` parameters.

Author: ```suggestion
TIP: To set the admin username and password through the `ServiceTelemetry` object, use the `graphing.grafana.adminUser` and `graphing.grafana.adminPassword` parameters.
```
